docs/modules/ROOT/pages/whats-new.adoc

```diff
@@ -30,15 +30,27 @@ Before 5.5, Hazelcast clients could only have one of two types of connection to
 
 image:ROOT:client-routing.png[Hazelcast Cluster Routing diagram]
 
-For more info, see xref:clients:java.adoc#client-cluster-routing-modes[Client cluster routing modes].
+For more information, see xref:clients:java.adoc#client-cluster-routing-modes[Client cluster routing modes].
+
+== New {java-client-new} available (BETA)
+This release introduces the Hazelcast Java Client (Standalone) v5.5.0-BETA. This is a dedicated lightweight Java client designed for efficient interaction with Hazelcast clusters. This client is available separately to the Java Client and Embedded Server that is bundled as part of the Hazelcast Platform. The standalone client supports mostly the same functionality as the Java Client and Embedded Server available with Hazelcast Platform but with less dependencies.
```

Comment:
```suggestion
This release introduces the {java-client-new} v5.5.0-BETA. This is a dedicated lightweight Java client designed for efficient interaction with Hazelcast clusters. This client is available separately to the Java Client and Embedded Server that is bundled as part of the Hazelcast Platform. The standalone client supports mostly the same functionality as the Java Client and Embedded Server available with Hazelcast Platform but with less dependencies.
```

Reply:
Good point but this section coming out for now (I copied from GH repo rather than AD source, hence the var disappearing)